src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -70,12 +70,12 @@ namespace OpenMS
   {
 public:
 
-    inline static ModificationsDB * getInstance()
+    inline static ModificationsDB * getInstance(OpenMS::String unimod_file = "CHEMISTRY/unimod.xml", OpenMS::String psimod_file = "CHEMISTRY/PSI-MOD.obo")
     {
       static ModificationsDB * db_ = 0;
       if (db_ == 0)
       {
-        db_ = new ModificationsDB;
+        db_ = new ModificationsDB(unimod_file, psimod_file);
```

Comment:
Yes, that's a good point. I also think that it is potentially dangerous, especially by having this implemented by default. Should we better consider one of the other options?